### PR TITLE
Rework Layered Attestation Environments intro

### LINKED ISSUE #230: obscure paragraph  "Layered Attestation Environments" section
I don't understand what this para is supposed to communicate:
```
   By definition, the Attester role creates Evidence.  An Attester may
   consist of one or more nested or staged environments, adding
   complexity to the architectural structure.  The unifying component is
   the root of trust and the nested, staged, or chained attestation
   Evidence produced.  The nested or chained structure includes Claims,
   collected by the Attester to aid in the assurance or believability of
   the attestation Evidence.
```

Also, pick one of "nested", "staged" or "chained" and drop the rest.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Based on Henk's proposal and Dave's comments.

Fixes #230

### CODE REVIEW

**Review by mcr — CHANGES_REQUESTED** [review]
I prefer that each sentence start on a newline as it makes git diffs easier.

**Comment by thomas-fossati** [comment]
> I prefer that each sentence start on a newline as it makes git diffs easier.

@mcr : This is a complete re-write of the paragraph which needs to be read in its entirety.  I'm not sure whether a line-by-line diff is of any help?  

That said, if you really want I could do that easily.

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -407,15 +407,15 @@ Execution environments that are designed specifically to be capable of claims co
...
 An Attester may consist of one or
-more nested or staged environments, adding complexity to the architectural
-structure. 
-The unifying component is the root of trust and the nested, staged,
-or chained attestation Evidence produced. 
-The nested or chained structure
-includes Claims, collected by the Attester to aid in the assurance or
-believability of the attestation Evidence.
+more nested environments (layers).
+The root layers of an Attester are called roots of trust.
```
Comment (henkbirkholz) [inline draft-ietf-rats-architecture.md:413]:
```suggestion
The root layer of an Attester encompasses the initial roots of trust included in an Attester
```

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -407,15 +407,15 @@ Execution environments that are designed specifically to be capable of claims co
...
+In order to appraise Evidence produced by an Attester, the Verifier needs to trust the Attester's root of trust.
```
Comment (henkbirkholz) [inline draft-ietf-rats-architecture.md:414]:
```suggestion
In order to appraise Evidence generated by an Attester, the Verifier needs to trust the Attester's root of trust.
```

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -407,15 +407,15 @@ Execution environments that are designed specifically to be capable of claims co
...
+Trust in the Attester's root of trust can be established either directly (e.g., the Verifier puts the root of trust's public key into its trust anchor store) or transitively via an Endorser (e.g., the Verifier puts the Endorser's public key into its trust anchor store).
+In layered attestation, a root of trust is the initial Attesting Environment.
+Claims can be collected from or about each layer, including roots of trust.
```
Comment (henkbirkholz) [inline draft-ietf-rats-architecture.md:417]: "Claims can be collected from or about each layer, including roots of trust."

This is technically correct. On the glossary level we have the responsibility to make absolutely sure that the reader is aware that a root of trust cannot ever generate Evidence from Claims about itself.

**Comment by mcr** [comment]
Thomas Fossati <notifications@github.com> wrote:
    > @mcr : This is a complete re-write of the paragraph which needs to be
    > read in its entirety.  I'm not sure whether a line-by-line diff is of
    > any help?

It helps with the next edit that comes along.

**Comment by thomas-fossati** [comment]
> Thomas Fossati <notifications@github.com> wrote: @mcr : This is a complete re-write of the paragraph which needs to be read in its entirety. I'm not sure whether a line-by-line diff is of any help? It helps with the next edit that comes along.

OK, done!

**Review by mcr — APPROVED** [review]

**Review by nedmsmith — APPROVED** [review]

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -407,17 +407,15 @@ Execution environments that are designed specifically to be capable of claims co
...
-a nested fashion that reflects the nesting of the Attester's layers.
+By definition, the Attester role generates Evidence.
+An Attester may consist of one or
+more nested environments (layers).
+The root layers of an Attester are called roots of trust.
+In order to appraise Evidence produced by an Attester, the Verifier needs to trust the Attester's root of trust.
+Trust in the Attester's root of trust can be established either directly (e.g., the Verifier puts the root of trust's public key into its trust anchor store) or transitively via an Endorser (e.g., the Verifier puts the Endorser's public key into its trust anchor store).
+In layered attestation, a root of trust is the initial Attesting Environment.
+Claims can be collected from or about each layer, including roots of trust.
+The corresponding claims can be structured in a nested fashion that reflects the nesting of the Attester's layers.
```
Comment (nedmsmith) [inline draft-ietf-rats-architecture.md:419]: Normally, claims are not self-asserted by a layer or root of trust.

**Review by nedmsmith — CHANGES_REQUESTED** [review]

**Review by dthaler — APPROVED** [review]

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -407,15 +407,15 @@ Execution environments that are designed specifically to be capable of claims co
...
-or chained attestation Evidence produced. 
-The nested or chained structure
-includes Claims, collected by the Attester to aid in the assurance or
-believability of the attestation Evidence.
+more nested environments (layers).
+The root layer of an Attester includes at least one root of trust.
+In order to appraise Evidence generated by an Attester, the Verifier needs to trust the Attester's root of trust.
+Trust in the Attester's root of trust can be established either directly (e.g., the Verifier puts the root of trust's public key into its trust anchor store) or transitively via an Endorser (e.g., the Verifier puts the Endorser's public key into its trust anchor store).
+In layered attestation, a root of trust is the initial Attesting Environment.
+Claims can be collected from or about each layer, including roots of trust.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:417]:
```suggestion
Claims can be collected from or about each layer.
```

**Pull request merged**